Ladybird/Tab.cpp

```diff
@@ -667,6 +667,17 @@ void Tab::show_console_window()
         m_console_widget = new Ladybird::ConsoleWidget;
         m_console_widget->setWindowTitle("JS Console");
         m_console_widget->resize(640, 480);
+
+        // Make the copy action available in the window via the bound copy key shortcut.
+        // Simply adding it to the context menu is not enough.
+        m_console_widget->addAction(&m_window->copy_selection_action());
+
+        m_console_context_menu = make<QMenu>("Context menu", m_console_widget);
```

Comment:
Tab is a QObject, yes? If so, we should use the Qt ownership model here, not OwnPtr.